services/sources/publisher_backlist_service.py

```diff
@@ -29,7 +30,10 @@ def __init__(self):
         self.es_manager = ElasticsearchManager()
         self.es_manager.createElasticConnection()
         
-        self.airtable_auth_token = os.environ.get('AIRTABLE_KEY', None)
+        if os.environ['ENVIRONMENT'] == 'production':
+            self.airtable_auth_token = get_parameter('arn:aws:ssm:us-east-1:946183545209:parameter/drb/production/airtable/pub-backlist/api-key')
+        else:
+            self.airtable_auth_token = get_parameter('arn:aws:ssm:us-east-1:946183545209:parameter/drb/qa/airtable/pub-backlist/api-key')
```

Comment:
I recommend just passing in the path `/airtable/pub-backlist/api-key` and the get_parameter function can figure out the environment and add the prefix!